documentation/book/proc-using-openshift-builds-create-image.adoc

```diff
@@ -4,28 +4,30 @@
 // assembly-deployment-configuration-kafka-connect-s2i.adoc
 
 [id='using-openshift-s2i-create-image-{context}']
-= Using {OpenShiftName} builds and S2I to create new images
+= Creating a container image using OpenShift builds
 
-{OpenShiftName} supports link:https://docs.openshift.org/3.9/dev_guide/builds/index.html[builds^], which can be used together with the link:https://docs.openshift.org/3.9/creating_images/s2i.html#creating-images-s2i[Source-to-Image (S2I)^] framework to create new container images.
-An {OpenShiftName} build takes a builder image with S2I support together with the connector plugin binaries provided by the user and uses them to build a new container image.
-The newly created container image is stored in {OpenShiftName}'s local container image repository and can be used in deployments.
+You can use {OpenShiftName} link:https://docs.openshift.org/3.9/dev_guide/builds/index.html[builds^] and the  link:https://docs.openshift.org/3.9/creating_images/s2i.html#creating-images-s2i[Source-to-Image (S2I)^] framework to create new container images. In an {OpenShiftName} build, a new container image is built from the following components:
 
-{ProductName} provides a Kafka Connect builder image, which can be found on {DockerRepository} as `{DockerKafkaConnectS2I}` with this S2I support.
-The S2I image takes user-provided binaries (with plugins and connectors), places them in the `/tmp/kafka-plugins/s2i` directory, and creates a new Kafka Connect image with them.
-This enhanced Kafka Connect image can be used with the Kafka Connect deployment.
-When Kafka Connect is started using the new image, it will automatically load all plugins or connectors from the `/tmp/kafka-plugins/s2i` directory.
+* A builder image with S2I support.
+
+* The binaries for your connector plug-in.
+
+Once built, new container images are stored in {OpenShiftName}'s local container image repository and are available for use in deployments.
+
+A Kafka Connect builder image with S2I support is provided by {ProductName} on the {DockerRepository} as `{DockerKafkaConnectS2I}`. This S2I image takes your binaries (with plug-ins and connectors) and stores them in the `/tmp/kafka-plugins/s2i` directory. It creates a new Kafka Connect image from this directory, which can then be used with the Kafka Connect deployment. When started using the enhanced image, Kafka Connect loads any third-party plug-ins from the `/tmp/kafka-plugins/s2i` directory.
 
 .Procedure
 
-. Create a Kafka Connect S2I cluster from the command-line
+To create a new container image using an {OpenShiftName} build:
```

Comment:
This is not what it does. What you do in this step is you deploy Kafka Connect with S2I support.